sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.1.0"
+VERSION = "0.2.0"
```

Comment:
Is there a reason we are keeping these versions started with "0"? @lmazuel is this a pattern we expect to continue seeing or should this be changed to `1.0.0b1`?